pkg/groups/managment.go

```diff
@@ -0,0 +1,8 @@
+package groups
```

Comment:
maybe better to move other groups to this module ?

Reply:
you are right, I will move them